src/task/methods/jn_ap_op2/script.py

```diff
@@ -424,7 +426,7 @@ def train(
 print('Generate predictions', flush=True)
 # ... generate predictions ...
 
-n_replica = 10 
+n_replica = par["id_map"] 
```

Comment:
Did you mean `par["epochs"`] instead? Also, feel free to change the name `--epochs` to something different, I just wrote that as an example.